<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>VtuberMusic</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI, Microsoft YaHei, PingFangSC-Regular, sans-serif, SimHei</family>
    <pointsize>10</pointsize>
    <weight>9</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>VRC Maker</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/logo_mid.png</normaloff>icon/logo_mid.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 10pt &quot;Microsoft YaHei UI, Microsoft YaHei, PingFangSC-Regular, sans-serif, SimHei&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>361</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 9pt &quot;PingFangSC-Regular, sans-serif, SimHei&quot;;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>180</y>
      <width>371</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 9pt &quot;PingFangSC-Regular, sans-serif, SimHei&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>歌词原文</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>150</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>歌词翻译</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>提示：纯中文歌翻译可以没有，但请注意翻译的时间轴必须和
原文时间轴完全对应，否则转出的VRC将出现问题</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>导入LRC</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>导入LRC</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>导入一轴双语LRC</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>导入歌曲</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>时间退回2秒</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>插入时间轴</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_7">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>导出VRC</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_8">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>导出原文LRC</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>             打轴区未开发，暂不可使用</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_9">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>打开VRC文件</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>361</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_10">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>导出双语LRC</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_11">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>60</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>导出翻译LRC</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_12">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>抓取网易云歌词</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>导出</string>
     </property>
     <addaction name="action_VRC_2"/>
     <addaction name="action_LRC"/>
     <addaction name="action_out_LRC2"/>
     <addaction name="action_LRC_5"/>
    </widget>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>导入</string>
     </property>
     <addaction name="action_VRC_3"/>
     <addaction name="action_LRC_2"/>
     <addaction name="action_LRC_3"/>
     <addaction name="action_LRC_4"/>
    </widget>
    <addaction name="menu_3"/>
    <addaction name="menu_4"/>
    <addaction name="separator"/>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="action_VtuberMusic"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_4">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_VtuberMusic">
   <property name="text">
    <string>关于 VRC Maker</string>
   </property>
  </action>
  <action name="action_VRC_2">
   <property name="text">
    <string>导出VRC</string>
   </property>
  </action>
  <action name="action_LRC">
   <property name="text">
    <string>导出原文LRC</string>
   </property>
  </action>
  <action name="action_VRC_3">
   <property name="text">
    <string>导入VRC</string>
   </property>
  </action>
  <action name="action_LRC_2">
   <property name="text">
    <string>导入LRC到原文</string>
   </property>
  </action>
  <action name="action_LRC_3">
   <property name="text">
    <string>导入LRC到翻译</string>
   </property>
  </action>
  <action name="action_LRC_4">
   <property name="text">
    <string>导入双语LRC</string>
   </property>
  </action>
  <action name="action_out_LRC2">
   <property name="text">
    <string>导出翻译LRC</string>
   </property>
  </action>
  <action name="action_LRC_5">
   <property name="text">
    <string>导出双语LRC</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
